src/components/cylc/WarningIcon.vue

```diff
@@ -17,39 +17,67 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 <!--
     WarningIcon - A dismiss-able warning icon
+
+    states:
+    * no warnings: grey + translucent
+    * new warnings: yellow
```

Comment:
IMO messages > WARNING should be further distinguished - either by being red or (possibly too cryptic) containing an "!"